### DISCUSSION TITLE
dannychoinka@gmail.com

### DISCUSSION THREAD

**Discussion by REVOLUT2** [community/community#142566] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

J

**Comment by hamidconrade** [comment]
@REVOLUT2  Your question is quite vague. I am of the idea you should re-ask. You can find answers to common problems from the FAQs page and I have attatched the link here. Happy coding. https://github.com/frequently-asked-questions/

**Comment by ebndev** [comment]
Hi @REVOLUT2, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
